src/cloud/CMakeLists.txt

```diff
@@ -28,18 +28,28 @@ set(MODULE_SRC
     ${CMAKE_CURRENT_LIST_DIR}/cloudmodule.cpp
     ${CMAKE_CURRENT_LIST_DIR}/cloudmodule.h
     ${CMAKE_CURRENT_LIST_DIR}/iauthorizationservice.h
-    ${CMAKE_CURRENT_LIST_DIR}/icloudprojectsservice.h
     ${CMAKE_CURRENT_LIST_DIR}/icloudconfiguration.h
+    ${CMAKE_CURRENT_LIST_DIR}/icloudsregister.h
     ${CMAKE_CURRENT_LIST_DIR}/cloudtypes.h
     ${CMAKE_CURRENT_LIST_DIR}/clouderrors.h
-    ${CMAKE_CURRENT_LIST_DIR}/view/accountmodel.h
-    ${CMAKE_CURRENT_LIST_DIR}/view/accountmodel.cpp
-    ${CMAKE_CURRENT_LIST_DIR}/internal/cloudservice.h
-    ${CMAKE_CURRENT_LIST_DIR}/internal/cloudservice.cpp
-    ${CMAKE_CURRENT_LIST_DIR}/internal/cloudconfiguration.h
-    ${CMAKE_CURRENT_LIST_DIR}/internal/cloudconfiguration.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/view/cloudsmodel.h
+    ${CMAKE_CURRENT_LIST_DIR}/view/cloudsmodel.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/internal/cloudsregister.h
+    ${CMAKE_CURRENT_LIST_DIR}/internal/cloudsregister.cpp
     ${CMAKE_CURRENT_LIST_DIR}/internal/oauthhttpserverreplyhandler.h
     ${CMAKE_CURRENT_LIST_DIR}/internal/oauthhttpserverreplyhandler.cpp
+
+    ${CMAKE_CURRENT_LIST_DIR}/internal/abstractcloudservice.h
+    ${CMAKE_CURRENT_LIST_DIR}/internal/abstractcloudservice.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/musescorecom/imusescorecomcloudservice.h
+    ${CMAKE_CURRENT_LIST_DIR}/musescorecom/musescorecomcloudservice.h
+    ${CMAKE_CURRENT_LIST_DIR}/musescorecom/musescorecomcloudservice.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/audiocom/iaudiocomcloudservice.h
+    ${CMAKE_CURRENT_LIST_DIR}/audiocom/audiocomcloudservice.h
+    ${CMAKE_CURRENT_LIST_DIR}/audiocom/audiocomcloudservice.cpp
```

Comment:
I don't insist, but audiocomservice and musescorecomservice (without the word "cloud") look simpler